<launch>

    <node pkg="odometry" type="odometry_node" name="odometry_node" output="screen">
        
        <param name="active" value="true"/>
        <param name="publish" value="true"/>
        <param name="twist_topic" value="/Topoistion"/>
        <param name="odom_topic" value="/odom"/>
        <param name="fixed_frame" value="odom"/>
        <param name="target_frame" value="base_footprint"/>
        <param name="update_params" value="false"/>

    </node>

</launch>
